### ISSUE TITLE
streaming.py:  SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by umentu** [tweepy/tweepy#1064]
```
Traceback (most recent call last):
  File "main.py", line 7, in <module>
    import tweepy
  File "/home/ubuntu/.venv/watson/lib/python3.7/site-packages/tweepy/__init__.py", line 17, in <module>
    from tweepy.streaming import Stream, StreamListener
  File "/home/ubuntu/.venv/watson/lib/python3.7/site-packages/tweepy/streaming.py", line 358
    def _start(self, async):
                         
SyntaxError: invalid syntax
```

async is a reserved word from 3.7.0

https://docs.python.org/3/whatsnew/3.7.html

**Comment by fitnr** [comment]
This issue is a duplicate.

**Comment by Harmon758** [comment]
Specifically, of #1017 and fixed with #1042.
